### PR TITLE
Add verticalMultiline.atDefnSite to .scalafmt.config

### PR DESCRIPTION
**Pull request by toddburnside**
This resulted in a lot of minor formatting changes.

There was only one similar to what I had in tmp-api (Restorer.scala), which I think is a big improvement. 

Other than that, I think some of the changes are a little better, some are a little worse, and some are just different. I have no strong opinion about whether it is an overall improvement or if we should make the change.

### CODE REVIEW

**Review by cquiroz — APPROVED** [review]
I think I like it

File: common/src/main/scala/explore/common/UserPreferencesQueries.scala [hunk 1]
```diff
@@ -106,7 +106,8 @@ object UserPreferencesQueries {
       userId:       Option[User.Id],
       area:         ResizableSection,
       defaultValue: Int
-    )(implicit cl:  GraphQLClient[F, UserPreferencesDB]): F[Int] =
+    )(implicit cl:  GraphQLClient[F, UserPreferencesDB]
+    ): F[Int] =
```
Comment (rpiaggio) [inline common/src/main/scala/explore/common/UserPreferencesQueries.scala:110]: Ouch

Comment (toddburnside) [reply common/src/main/scala/explore/common/UserPreferencesQueries.scala:110]: Yeah, this falls into the "some are a little worse" category. I do like the other places where it puts the final case class closing paren on the same line as the last parameter, though. 🤷

**Review by rpiaggio — COMMENTED** [review]
I do like:
* That all parameters in all parameter lists are aligned.

I don't like:
* That the closing parenthesis is moved to the line of the last parameter. When it was in the next line, it nicely delimited visually the end of the parameter block.

I definitely don't like:
* That single-parameter lists are not closed in the same line.

In other words, it seems the closing parenthesis at the end of single- and multi-parameter lists behave exactly the reverse as I would expect (which would be same line for single- and next line for multi-).

**Comment by toddburnside** [comment]
>     * That the closing parenthesis is moved to the line of the last parameter. When it was in the next line, it nicely delimited visually the end of the parameter block.

I agree that it is a little less clear with `continuationIndet.defnSite = 2`. I think that is why the default for that parameter is 4 - to visually delimit the end of the parameter block. It would also align the parameters up with `extends`. But it's all a matter of taste.

Maybe we should just stick with what we have and use `// format: off` for egregious cases?

**Comment by cquiroz** [comment]
I really wish dotty comes with a std conf for scalafmt that everybody uses

**Comment by toddburnside** [comment]
> I really wish dotty comes with a std conf for scalafmt that everybody uses

😆 That is what the default configuration for scalafmt was supposed to be - but nobody could agree.

**Comment by rpiaggio** [comment]
Sorry for the long silence. Maybe we can play a bit more with the other `verticalMultiline` options? Let me see if I can get to a configuration that addresses all our woes and if that happens I'll send a PR to this PR.

**Comment by toddburnside** [comment]
> Sorry for the long silence. Maybe we can play a bit more with the other `verticalMultiline` options? Let me see if I can get to a configuration that addresses all our woes and if that happens I'll send a PR to this PR.

No worries. As I said above, I'm not strongly attached to this. I'm just glad we have something.

**Comment by toddburnside** [comment]
Will be replaced with a different PR.

**Pull request closed without merging**